src/evict/evict_lru.c

```diff
@@ -75,7 +75,8 @@ __evict_entry_priority(WT_SESSION_IMPL *session, WT_REF *ref)
 		return (WT_READGEN_OLDEST);
 
 	/* Any page from a dead tree is a great choice. */
-	if (F_ISSET(btree->dhandle, WT_DHANDLE_DEAD))
+	if (F_ISSET(btree->dhandle, WT_DHANDLE_DEAD) ||
+	    F_ISSET(btree, WT_BTREE_LOOKASIDE))
```

Comment:
We want lookaside pages out of cache as soon as possible.  There is no fast path here -- the application doesn't want to access lookaside pages, it wants the cache to not be full so it can make progress with real work.  That change has been split out into a separate ticket.